vnext/src/overrides.json

```diff
@@ -3,6 +3,30 @@
     "tsconfig.json"
   ],
   "overrides": [
+    {
+      "type": "patch",
```

Comment:
FYI this whole file is about to move with a PR I have out right now

Reply:
yeah, I saw.

Reply:
I'll let your change get in first.

Reply:
Just merged it in, so you should be able to merge now.